src/hotspot/share/opto/convertnode.hpp

```diff
@@ -100,6 +100,18 @@ class ConvF2DNode : public Node {
   virtual uint  ideal_reg() const { return Op_RegD; }
 };
 
+//------------------------------ConvF2HFNode------------------------------------
+// Convert Float to Halffloat
+class ConvF2HFNode : public Node {
+  public:
+  ConvF2HFNode( Node *in1 ) : Node(0,in1) {}
```

Comment:
Additional space b/w , and in1

Reply:
Hi Jatin, this spacing is similar to other class declarations in the file. So I will keep it consistent. Thanks.